client/actions/form.js

```diff
@@ -35,11 +35,11 @@ export async function fetchAppointments() {
     name: item.name,
     status: item.status ? item.status : "Requested",
     dateCreated: new Date(item.dateCreated),
-    timeRange: item.timeRange,
+    timeRange: item.preferredTimeRange,
     phone: item.phone,
     email: item.email,
-    address: item.address,
-    location: item.location,
+    address: item.location.address,
```

Comment:
I think this is because you have some docs in your appointments collection that are using the older Appointment model, so `location.address` does not exist

Reply:
Ill try removing them and test again!